README.md

```diff
@@ -18,11 +18,57 @@ In your `Podfile`, add the following entry:
 pod 'ChartboostMediationAdapterReference'
 ```
 
-## Chartboost Mediation Custom Adapter Implementation Guidelines
+## Chartboost Mediation Custom Adapter Implementation Guide
 
 > [!IMPORTANT]
 > Chartboost Mediation will not be providing official support for any custom adapters. For official adapters that we develop and support, [visit this site](https://adapters.chartboost.com).
 
+1. Create a new class and conform to Helium’s `PartnerAdapter` protocol.
+2. Implement var `partnerSDKVersion: String { get }` with the version of the partner SDK. Most adapters derive this from the partner SDKs API so that it always reports the correct number, even when a newer version is imported.
```

Comment:
```suggestion
2. Implement `var partnerSDKVersion: String { get }` with the version of the partner SDK. Most adapters derive this from the partner SDKs API so that it always reports the correct number, even when a newer version is imported.
```